### PR TITLE
[11.0][MIG] purchase_open_qty

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Superseeds https://github.com/OCA/purchase-workflow/pull/554.

### CODE REVIEW

File: purchase_open_qty/models/purchase_order.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                                  copy=False,
+                                  string="Qty to Receive", store=True)
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    def _compute_qty_to_invoice(self):
+        for po in self:
+            po.qty_to_invoice = sum(po.mapped('order_line.qty_to_invoice'))
```
Comment (nikul-serpentcs) [inline purchase_open_qty/models/purchase_order.py:56]: @mreficent IMO here Direct add `po.qty_to_receive = sum(po.mapped('order_line.qty_to_receive'))`

and remove `_compute_qty_to_receive`

File: purchase_open_qty/models/purchase_order.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            [('qty_to_receive', operator, value)])
+        order_ids = po_lines.mapped('order_id')
+        res.append(('id', 'in', order_ids.ids))
+        return res
+
+    qty_to_invoice = fields.Float(compute='_compute_qty_to_invoice',
+                                  search='_search_qty_to_invoice',
+                                  string="Qty to Bill",
+                                  default=0.0)
+    qty_to_receive = fields.Float(compute='_compute_qty_to_receive',
```
Comment (nikul-serpentcs) [inline purchase_open_qty/models/purchase_order.py:86]: here add `compute='_compute_qty_to_invoice',`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by mreficent** [comment]
@lreficent

File: purchase_open_qty/models/purchase_order.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply purchase_open_qty/models/purchase_order.py:56]: they have different api.depends fields.

File: purchase_open_qty/models/purchase_order.py (hunk 2 above)
Comment (LoisRForgeFlow) [reply purchase_open_qty/models/purchase_order.py:86]: same as above

**Comment by LoisRForgeFlow** [comment]
@nikul-serpentcs can you update your review?

**Review by LoisRForgeFlow — APPROVED** [review]
thanks

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza I think the approvals of https://github.com/OCA/purchase-workflow/pull/554 also count for this PR, right? 😉

**Comment by pedrobaeza** [comment]
But check CIs state

**Comment by MiquelRForgeFlow** [comment]
CI state was green but for an unknown reason now runbot is going stupid and show "503 Service Unavailable."

**Comment by MiquelRForgeFlow** [comment]
@pedrobaeza now is green the CI :green_apple:

**Pull request merged**